src/graphene-ray.c

```diff
@@ -53,8 +53,9 @@
 #include <float.h>
 
 #if defined(_M_X64) && ! defined(isnanf)
```

Comment:
You are piggybacking a special case instead of handling it the same way as before. This goes against what you suggested in https://github.com/ebassi/graphene/pull/225#issuecomment-817358342.

This should be:
```
#if defined(_M_X64) && !defined(isnanf)
# define my_isnan(x) _isnanf(x)
#elif HAVE_ISNANF
# define my_isnan(x) isnanf(x)
#else
# define my_isnan(x) isnan(x)
#endif
```

Reply:
Thanks, I had only grepped for `HAVE_ISNANF`. I have now seen https://github.com/ebassi/graphene/blob/master/meson.build#L172